modules/test/playwright/tests/frontend-data-set-views-web/utils/types.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
- We usually have a `types` folder for typing related things. We could start with a shared `index.d.ts`